src/client/containers/ProfilePage/AdminProfile/AdminProfile.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+
+import admin from '../../../assets/images/admin.png';
+import { Layout } from '../../../components';
+import { AppHeader } from '../../../components/Appheader/AppHeader.component';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import Footer from '../../../components/footer/footer';
+import Header from '../../../components/Heading/Heading';
+import Input from '../../../components/Input/Input';
+import { useUser } from '../../../context/userContext';
+import { useAuthenticatedFetch } from '../../../hooks/useAuthenticatedFetch';
+
+export const AdminProfilePage = () => {
+  const {
+    user: { name },
+  } = useUser();
+  const [adminRole, setAdminRole] = useState(null);
+  const { fetch } = useAuthenticatedFetch();
+  useEffect(() => {
+    fetch('/api/profile/admin').then((data) => {
+      setAdminRole(data[0].adminRole);
```

Comment:
Why `[0]` here? The endpoint returns a profile - which means single profile. Not profiles (plural).

Reply:
endpoint returns single profile as array of object [{}], so `[0]`.

Reply:
You don't return a single profile as an array - that makes no sense. It is conventional to return an object from an API endpoint that matches the resource.

Reply:
i will change endpoint